### ISSUE TITLE
Not compatible with latest BinDeps version 1.0.0?

### ISSUE THREAD

**Issue by JianghuiDu** [JuliaPackaging/CondaBinDeps.jl#2]
```
(v1.3) pkg> add CondaBinDeps
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package CondaBinDeps [a9693cdc]:
 CondaBinDeps [a9693cdc] log:
 ├─possible versions are: 0.1.0 or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions 0.1.0
 └─restricted by compatibility requirements with BinDeps [9e28174c] to versions: uninstalled — no versions left
   └─BinDeps [9e28174c] log:
     ├─possible versions are: [0.7.0, 0.8.9-0.8.10, 1.0.0] or uninstalled
     └─restricted to versions 1.0.0 by an explicit requirement, leaving only versions 1.0.0
```

**Comment by Alexander-Barth** [comment]
I think the upper version bound is set the JuliaRegistries:

https://github.com/JuliaRegistries/General/blob/master/C/CondaBinDeps/Compat.toml

**Comment by JianghuiDu** [comment]
OK. Manually changed that on my computer. Now it works. Thanks!

**Comment by stevengj** [comment]
Closed by #3
